### ISSUE TITLE
Ditto Not Starting in Windows 7 32B Hyper-V VM

### ISSUE THREAD

**Issue by nerdx86** [sabrogden/Ditto#28]
Ditto Not Starting in Windows 7 32B Hyper-V VM

Is there any log, etc that I can look at?  It was working earlier today but now it fails to start..  The hot key doesn't work and it doesn't appear on the taskbar.   Tried Portable and regular versions..  Tried removing everything from drive AND registry, etc..

**Comment by sabrogden** [comment]
Are you up to date on all windows updates?  Last I tried on windows 7 it worked.  Downloading a windows 7 VM now to verify.

**Comment by nerdx86** [comment]
It was working, so I'm guessing that I corrupted something.  (That's why I
was asking about logs..)  It's windows 7x32, windows 7x64 works fine..  I
have a system internals procman log that I'm attaching..  Seems to hang at
the same spot in both "installed" and "portable" versions.  (which is a
"startthread" call and then nothing further, like a deadlock occurs or
something)  I think that it could be related to windows defender/firewall
but I haven't been able to nail that down either) (It stopped working
around the time that I reset windows defender's settings)

I originally was trying to have all my Dittos on my various VM's pointing
at the same DB, but that seemed to cause issues. (They were all working
then though, just required a refresh to move a clipboard from one VM to
another)  I switched to having them all based on "friends" with them trying
to send to all the VM's.

My entire use case here is multiple isolated development environments
running under hyper-V so I want to be able to copy/paste between them.
(windows 7/xp don't carry the clipboard buffers from hyper-V)

Another option for me would be just to create my own "app" that uses a
shared drive and moves the data back and forth, but a coworker swears by
your tool (for like 15 years or something).. :)
